src/services/notificationHandler.js

```diff
@@ -0,0 +1,8 @@
+'use strict'
+
+const NotificationHandler = require('../lib/notificationHandlerWebsocket')
```

Comment:
Out of curiosity, what's the point of changing `notificationWorker` to `notificationHandler`? (Handler kind of sounds to me like something that receives notifications, not something that sends them out)